packages/date-picker/src/FilterDateRangePicker/components/DateRangeInputField/DateRangeInputField.module.scss

```diff
@@ -0,0 +1,45 @@
+@import "~@kaizen/design-tokens/sass/border";
+
+$input-disabled-opacity: 0.3;
+
+.dateRangeInputContainer {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  border: none;
+}
+
+.inputRangeStart {
+  border-start-end-radius: 0;
+  border-end-end-radius: 0;
+
+  &:focus + div {
+    border-start-end-radius: 0;
+    border-end-end-radius: 0;
+  }
+}
+
+.inputRangeEnd {
+  position: relative;
+  inset-inline-start: calc(-1 * #{$border-solid-border-width});
+
+  border-start-start-radius: 0;
+  border-end-start-radius: 0;
+
+  &:focus + div {
```

Comment:
Nit pick: these could be applied without the `:focus` part (I think).

Also, obviously the element reference here is prone to regressing, if someone were to change that div to a span with no knowledge of this separate component referencing this. You could do a wildcard selector for the class name "focus ring", which is still not ideal but a bit less prone to changing.

```
+ [class*="focusRing"] {
```

Reply:
I think ideally we'd refactor the Input component so I don't even need to do something horrible like this 😂 But yeah I suppose that's something that could be done 😬